console-subscriber/README.md

```diff
@@ -206,6 +206,10 @@ tracing_subscriber::registry()
 [`tokio-console`]: https://github.com/tokio-rs/console
 [Tokio]: https://tokio.rs
 
+### Using other runtimes
+
+If you are using a custom runtime that supports tokio-console, you may not need to enable "tokio_unstable". In this case, you need to enable cfg `console_without_tokio_unstable` for console-subscriber to disable its check for `tokio_unstable`.
```

Comment:
do you mind wrapping this line to match the remaining lines in this file? Thank you!

Reply:
Sorry, haven't notice it. Fixed